components/TextFields/examples/TextFieldSemanticColorThemer.swift

```diff
@@ -0,0 +1,259 @@
+                                       constant: 0)]
+    constraints += [NSLayoutConstraint(item: textfieldStandard,
+                                       attribute: .trailing,
+                                       relatedBy: .equal,
+                                       toItem: view,
+                                       attribute: .trailingMargin,
+                                       multiplier: 1,
+                                       constant: 0)]
+
+    #if swift(>=3.2)
+    if #available(iOS 11.0, *) {
+      constraints += [NSLayoutConstraint(item: textfieldStandard,
+                                         attribute: .top,
+                                         relatedBy: .equal,
+                                         toItem: scrollView.contentLayoutGuide,
+                                         attribute: .top,
+                                         multiplier: 1,
+                                         constant: 20),
+                      NSLayoutConstraint(item: textfieldAlternative,
+                                         attribute: .bottom,
+                                         relatedBy: .equal,
+                                         toItem: scrollView.contentLayoutGuide,
+                                         attribute: .bottomMargin,
+                                         multiplier: 1,
+                                         constant: -20)]
+    } else {
+      constraints += [NSLayoutConstraint(item: textfieldStandard,
+                                         attribute: .top,
+                                         relatedBy: .equal,
+                                         toItem: scrollView,
+                                         attribute: .top,
+                                         multiplier: 1,
+                                         constant: 20),
+                      NSLayoutConstraint(item: textfieldAlternative,
+                                         attribute: .bottom,
+                                         relatedBy: .equal,
+                                         toItem: scrollView,
+                                         attribute: .bottomMargin,
+                                         multiplier: 1,
+                                         constant: -20)]
+    }
+    #else
+    constraints += [NSLayoutConstraint(item: textfieldStandard,
+    attribute: .top,
+    relatedBy: .equal,
+    toItem: scrollView,
+    attribute: .top,
+    multiplier: 1,
+    constant: 20),
+    NSLayoutConstraint(item: textfieldAlternative,
+    attribute: .bottom,
+    relatedBy: .equal,
+    toItem: scrollView,
+    attribute: .bottomMargin,
+    multiplier: 1,
+    constant: -20)]
+    #endif
+
+    NSLayoutConstraint.activate(constraints)
+  }
+
+  func setupScrollView() {
+    view.addSubview(scrollView)
+    scrollView.translatesAutoresizingMaskIntoConstraints = false
+
+    NSLayoutConstraint.activate(NSLayoutConstraint.constraints(
```

Comment:
please stay consistent with the indentation of the one below

Reply:
C&P Formatting fixed.